server/src/main/java/org/opensearch/common/util/concurrent/ThreadContext.java

```diff
@@ -215,6 +217,17 @@ public Writeable captureAsWriteable() {
      * if it can't find the task in memory.
      */
     public StoredContext stashWithOrigin(String origin) {
```

Comment:
Merging this one, we could backport javadocs later on (https://github.com/opensearch-project/OpenSearch/pull/15039#discussion_r1699091954)